packages/replayer/src/replay.utils.ts

```diff
@@ -148,49 +148,60 @@ export const patchDate: (options: {
   `);
 };
 
-const getAppUrl: (options: {
-  sessionData: any;
+export const getStartUrl: (options: {
```

Comment:
There is one change in the logic here.

There are two recorded start URLs -- sessionData.userEvents.window.startUrl is the URL after 1500ms, while session.startUrl is the URL straight away.

Previously if the appUrl was falsey and `session.startUrl` is truthy, then it'd use `sessionData.userEvents.window.startUrl` if `sessionData.userEvents.window.startUrl` was more than just an origin, and `session.startUrl` with the origin of `sessionData.userEvents.window.startUrl` otherwise. I'm pretty sure this was unintended / a bug.

Now if appUrl is falsey and `session.startUrl` is truthy then it'll use `session.startUrl`